balboa-client/src/main/scala/com/socrata/metrics/collection/SocrataQueues.scala

```diff
@@ -2,8 +2,6 @@ package com.socrata.metrics.collection
 
 import java.util.concurrent.{LinkedBlockingQueue, TimeUnit}
 
-import scala.collection.mutable.Queue
-
 trait SocrataAbstractQueue[Q] {
```

Comment:
We could just take out SocrataAbstractQueue and have PreBufferQueue store a BlockingQueue directly.